### DISCUSSION TITLE
redev-2012/

### DISCUSSION THREAD

**Discussion by martijnboland** [martijnboland/blog#46] [Blog comments]
# redev-2012/

https://blogs.taiga.nl/martijn/posts/redev-2012/

**Comment by disqus-comments-migrator** [comment]
<em>René van den Berg commented [on Disqus](https://disqus.com/home/forum/martijnboland/) at november 15 2012, 02:14</em>

---

<p>Very interesting. Sounds like this event is very inspiring.</p>
